src/modules/PositionModule.cpp

```diff
@@ -338,7 +382,7 @@ void PositionModule::sendOurPosition()
         if (channels.getByIndex(channelNum).settings.has_module_settings &&
             channels.getByIndex(channelNum).settings.module_settings.position_precision != 0) {
             sendOurPosition(NODENUM_BROADCAST, requestReplies, channelNum);
-            return;
```

Comment:
We deliberately do not periodically transmit position out over multiple channels to avoid a storm of packets, only the first channel that has position enabled is used. The other channels are request-only.

Reply:
> The other channels are request-only.

Can you help me find the doc how to do other channels request-only? So far, using Android, I only see "exchange position" when viewing details of another node. I assume the precision for that "exchange position" is the setting of public channel.

> We deliberately do not periodically transmit position out over multiple channels to avoid a storm of packets, only the first channel that has position enabled is used.

Can this be made configurable, or is it a hardcoded design choice? If it’s hardcoded, I assume nodes running a patched firmware (that removes that return statement) could still broadcast position on multiple channels, correct?